crossing-training.lic

```diff
@@ -60,6 +60,13 @@ class CrossingTraining
     Flags.add('research-complete', '^Breakthrough!')
   end
 
+  def dump_junk
+    if Time.now - @junk_timer > 300
```

Comment:
I'd prefer to see this conditional inverted and used as a guard clause (so `return if ...`). What do you think?